gcloud-java-datastore/src/main/java/com/google/gcloud/datastore/EntityQuery.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2015 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.gcloud.datastore;
+
+/**
+ * An implementation of a Google Cloud Datastore entity query that can be constructed by providing
+ * all the specific query elements.
```

Comment:
It might be due to the fact that I am not very much into datastore but this definition sounds a bit foggy to me. What are "all the specific query elements"?

This comment also applies to `KeyQuery` and `ProjectionEntityQuery`.